framework/db/Command.php

```diff
@@ -736,7 +736,110 @@ public function dropIndex($name, $table)
     {
         $sql = $this->db->getQueryBuilder()->dropIndex($name, $table);
 
-        return $this->setSql($sql);
+        return $this->setSql($sql)->requireTableSchemaRefresh($table);
+    }
+
+    /**
+     * Creates a SQL command for adding an unique constraint to an existing table.
+     * @param string $name the name of the unique constraint.
+     * The name will be properly quoted by the method.
+     * @param string $table the table that the unique constraint will be added to.
+     * The name will be properly quoted by the method.
+     * @param string|array $columns the name of the column to that the constraint will be added on.
+     * If there are multiple columns, separate them with commas.
+     * The name will be properly quoted by the method.
+     * @return $this the command object itself.
+     * @since 2.0.13
+     */
+    public function addUnique($name, $table, $columns)
```

Comment:
This migration enchantments cover almost all usable constraints.

I think it _may_ be worth to keep uniques as well as it can be supported (I mean a distinction between unique constraints and unique indexes) in next versions or may be handy when extending core classes.

But I'm not sure. I still can't way any (useful in PHP) db which doesn't create an index behind the constraint. (NoSQL?..)

I would like to hear opinions if it should go or stay.